src/test/unit/osd_unittest.cc

```diff
@@ -1242,6 +1242,7 @@ extern "C" {
     bool pidOsdAntiGravityActive(void) { return false; }
     bool failsafeIsActive(void) { return false; }
     bool gpsRescueIsConfigured(void) { return false; }
+    bool gpsIsHealthy(void) { return true; }
```

Comment:
If you have some time, it will be great to have two tests, one for healthy and the other one for not healthy. In this way the NC will be tested.

Reply:
I've created a simple test for the Sats element but it's not working and I can't figure why. The OSD_GPS_SATS element is supposed to be active and visible, but for whatever reason, osdElementGpsSats is not called and hence the test fails.

Reply:
I think I've found your problem:

https://github.com/betaflight/betaflight/blob/2a5e457603a94efa0a18df2eb733ccc0ef0b1c1c/src/main/osd/osd_elements.c#L1882-L1892

The GPS sats are blinking and you are verifying when hide. If I comment this blinking it works.

Reply:
If I change the numSats of your first test to 5 or more it works. What I don't have look is how to fix it "in a correct way".

Reply:
Ah, the blinking, you're right. I'll look if there's any specific way to test if it's actually blinking, which it should. Thanks!